pkg/exporter/azureblob_exporter_test.go

```diff
@@ -0,0 +1,27 @@
+package exporter
+
+import (
+	"testing"
+)
+
+var getStorageContainerNameTests = []struct {
+	apiServerFqdn string
+	containerName string
+}{
+	{"dakydd-test-eastus-dns-d0daedb9.hcp.eastus.azmk8s.io", "dakydd-test-eastus-dns-d0daedb9"},
```

Comment:
this test confirms that current behaviour of azureblob_exporter remains unchanged